src/storage/src/hummock/store/version.rs

```diff
@@ -287,24 +332,141 @@ impl HummockReadVersion {
                 let max_committed_epoch = committed_version.max_committed_epoch();
                 self.committed = committed_version;
 
-                {
-                    // TODO: remove it when support update staging local_sst
+                let table_id = self.table_id;
```

Comment:
This code is too complex. How about move them to a function.

Reply:
> This code is too complex. How about move them to a function.

fixed